### DISCUSSION TITLE
Enummer: Multi-value enums for Rails

### DISCUSSION THREAD

**Discussion by shkm** [therubyradar/submissions#40] [Submissions]
I released [Enummer](https://github.com/shkm/enummer) recently for what I'm calling "multi-value enums" aka flags.

With a similar API sto `enum`, it uses bitfields to store multiple values in the database under a single column. It could be considered a lightweight alternative to `flag_shih_tzu`.

I'm looking to expand the userbase beyond just myself to improve its robustness.

**Comment by andrewmcodes** [comment]
Looks neat!
